apps/src/calc/token.js

```diff
@@ -83,9 +83,11 @@ Token.prototype.setStringRepresentation_ = function () {
     return;
   }
 
+  // use toLocaleString so that we get commas per thousands
+
   // at this point we know we have a jsnumber
   if (this.val_.isInteger()) {
-    this.nonRepeated_ = this.val_.toFixnum().toString();
+    this.nonRepeated_ = this.val_.toFixnum().toLocaleString('en-US');
```

Comment:
Is toFixnum returning a jsnum? If not, is it worth testing [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) in IE9 / iOS / Firefox?